pkg/templates/clusteradminrolebinding/template.go

```diff
@@ -0,0 +1,42 @@
+package clusteradminrolebinding
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"golang.stackrox.io/kube-linter/pkg/check"
+	"golang.stackrox.io/kube-linter/pkg/config"
+	"golang.stackrox.io/kube-linter/pkg/diagnostic"
+	"golang.stackrox.io/kube-linter/pkg/lintcontext"
+	"golang.stackrox.io/kube-linter/pkg/objectkinds"
+	"golang.stackrox.io/kube-linter/pkg/templates"
+	"golang.stackrox.io/kube-linter/pkg/templates/clusteradminrolebinding/internal/params"
+	rbacV1 "k8s.io/api/rbac/v1"
+)
+
+func init() {
+	templates.Register(check.Template{
+		HumanName:   "cluster-admin role binding",
+		Key:         "cluster-admin-role-binding",
+		Description: "Flag bindings of cluster-admin role to service account",
```

Comment:
Are you only catching service accounts here? Seems like you're catching binding to anything? Either update the description or the check.

Reply:
i updated the description. it catches all of the referenced objects under subjects including serviceaccounts, users, groups.